pages/special-projects/centralized-calendar/end-user.mdx

```diff
@@ -0,0 +1,75 @@
+import { Callout } from "nextra/components";
+
+# End User Guide
+
+<Callout type="info">
+  This page is intended for members of other ICS organization to provide information on how to
+  participate in the ics-centralized-calendar project.
+</Callout>
+
+{/* Include a breif introduction of what is Centralized Calendar Project is and steps to get a the required Google Calendar URL. */}
+
+## Introduction
+
+The `ics-centralized-calendar` project is an easy to use API meant to provide centralized access to all ICS calendars, offering options for selecting specific clubs and output formats (ICS or JSON).
```

Comment:
I think approaching this introduction from a more project / goal oriented mindset might be more suitable for this page.

Something that covers:
- Bring the ICS community closer
- Makes your events easier to discover
- Helps us promote ICS events better

I don't think explaining that this project is an API is necessarily useful for the target audience of this doc (other clubs).